guardian/manage-frontend
# Add diagnostic information page to the help centre

## What does this change?

This introduces a new page to the help centre, this page provides the user with information that would be useful to be passed onto the UH team 

Information included:

- device information
- browser information
- cookie information
- account information

The user is given the option to copy to clipboard to send to the UH team 



## How to test

a user should be able to go to https://manage.theguardian.com/help-centre/diagnostic-information

and see relevant information about their browser and they should be able to copy this to their clipboard



## How can we measure success?

Right now we're sending users to 4 different URLs to get information, the goal of this page is to save time and make the process more efficient



## Have we considered potential risks?

Added a new call to the members data API - we're not sure if this is best practice within a project

Page is only discoverable by being sent the link directly, which limits the surface area for issues



## Images

![image](https://github.com/guardian/manage-frontend/assets/123190660/8b4c2bb0-040a-4a65-989f-47b78e508ae9)




## Accessibility
